spiped/dispatch.c

```diff
@@ -166,15 +172,16 @@ callback_gotconn(void * cookie, int s)
  * forward secrecy.  If ${requirefps} is non-zero, require that both ends use
  * perfect forward secrecy.  Enable transport layer keep-alives (if applicable)
  * if and only if ${nokeepalive} is zero.  Drop connections if the handshake or
- * connecting to the target takes more than ${timeo} seconds.  If ${conndone}
- * is not NULL, set it to non-zero value when a connection closes.  Returns a
- * cookie which can be passed to dispatch_shutdown.
+ * connecting to the target takes more than ${timeo} seconds.  If ${only_one}
+ * is non-zero request a shutdown once the first connection is acccepted.  If
```

Comment:
I suggest:
"If ${only_one} is a non-zero**, then** ${conndone} is..."
"... every connection ended**, then** ${conndone}..."
The original grammar is not strictly speaking incorrect, but the comma and "then" makes it easier to skim the sentence.